sitetree/sitetreeapp.py

```diff
@@ -608,7 +608,13 @@ def get_tree_current_item(self, tree_alias):
             return None
 
         # urlquote is an attempt to support non-ascii in url.
-        current_url = urlquote(self.current_request.path)
+        path = self.current_request.path
+        if type(path) == str:
```

Comment:
Here `isinstance` should be used.

Reply:
This branching will cost us something, bearing in mind it's a busy method. To reduce the cost we should try to pick conversion we need (if we really need it) on import time (e.g. `partial` or something).